test_rclcpp/test/test_local_parameters.cpp

```diff
@@ -379,6 +404,7 @@ TEST(CLASSNAME(test_local_parameters, RMW_IMPLEMENTATION), set_parameter_if_not_
 int main(int argc, char ** argv)
 {
   // NOTE: use custom main to ensure that rclcpp::init is called only once
+  setvbuf(stdout, NULL, _IONBF, BUFSIZ);
```

Comment:
Please move this line one line higher. Otherwise it separates the comment line from the line it is commenting.